module/VuFind/src/VuFind/View/Helper/Root/HelpText.php

```diff
@@ -100,39 +82,29 @@ public function getWarnings()
      *
      * @return string|bool
      */
-    public function render($name, $context = null)
+    public function render($name, $context=[])
     {
-        // Set up the needed context in the view:
-        $this->contextHelper->__invoke($this->getView());
-        $oldContext = $this->contextHelper
-            ->apply(null === $context ? [] : $context);
-
         // Sanitize the template name to include only alphanumeric characters
         // or underscores.
         $safe_topic = preg_replace('/[^\w]/', '', $name);
```

Comment:
While we're refactoring this anyway, might be a good time to rename `$safe_topic` to `$safeTopic` to be more in line with normal VuFind conventions.